### PR TITLE
Fixing incorrectly migrated structured logs

### LINKED ISSUE #105101: Fix incorrectly migrated structured logs
#### What happened:

When working on enabling static analysis for structured logs on whole codebase https://github.com/kubernetes/kubernetes/issues/102439 we have noticed that there are multiple places that were incorrectly migrated. https://github.com/kubernetes/kubernetes/pull/103293#issuecomment-885028839
Before we enable static analysis we need to fix those places.

#### What you expected to happen:

Running logcheck should not detect any invalid log calls.

#### How to reproduce it (as minimally and precisely as possible):

Run klog logcheck script on kubernetes repo with `--allow-unstructured` flag. https://github.com/kubernetes/klog/tree/main/hack/tools/logcheck

Please use klog from master branch as we just fixed Exit* https://github.com/kubernetes/klog/issues/264

#### Anything else we need to know?:

/wg structured-logging

### PR DESCRIPTION
**Pull request by shivanshuraj1333**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Fixing incorrectly migrated logs

#### Which issue(s) this PR fixes:

Fixes https://github.com/kubernetes/kubernetes/issues/105101

#### Special notes for your reviewer:
In this PR, some of missing keys are added which are necessary in structured logging

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by shivanshu1333** [comment]
Used https://gist.github.com/shivanshu1333/6b8000749d9af8a17949ce20f5ed2598 with https://github.com/kubernetes/klog/tree/main/hack/tools/logcheck on Kubernetes repo

output:
```
Running structured logging static check on migrated packages
/Users/shivashr/opensource/prow/go/src/k8s.io/kubernetes/pkg/proxy/healthcheck/service_health.go:59:3: Additional arguments to ErrorS should always be Key Value pairs. Please check if there is any key or value missing.
/Users/shivashr/opensource/prow/go/src/k8s.io/kubernetes/pkg/controller/volume/pvcprotection/pvc_protection_controller.go:353:2: Additional arguments to InfoS should always be Key Value pairs. Please check if there is any key or value missing.
Please fix above failures. You can locally test via:
hack/verify-structured-logging.sh
```
Fixed structured logging migration in mentioned files

File: pkg/controller/volume/pvcprotection/pvc_protection_controller.go [hunk 1]
```diff
@@ -350,7 +350,7 @@ func (c *Controller) pvcAddedUpdated(obj interface{}) {
 		utilruntime.HandleError(fmt.Errorf("couldn't get key for Persistent Volume Claim %#v: %v", pvc, err))
 		return
 	}
-	klog.V(4).InfoS("Got event on PVC", key)
+	klog.V(4).InfoS("Got event on PVC", "key", key)
```
Comment (serathius) [inline pkg/controller/volume/pvcprotection/pvc_protection_controller.go:353]:
```suggestion
	klog.V(4).InfoS("Got event on PVC", "pvc", klog.KObj(pvc))
```

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**